### PR TITLE
refactor!: template update

### PR DESCRIPTION
**Pull request by nemanja-drobac**
### Summary

Refactor ISSUE templates for FOS IT delivery -> Azure Champions Team

#### Acceptance Criteria

- Generalize maintenance issue templates for type `Collection` and `Epic`

## To do

- [x] Generalize maintenance issue templates for type `Collection` and `Epic`
- [x] Refactor templates
- [x] Add `Acceptance Criteria` choice in types 👑 📇 and `task`

### CODE REVIEW

**Review by eikeskogen — COMMENTED** [review]
Looks good! 
Please change `b_goal.md`  
line `#11`: 
from `<Team_Name>` to `<team_name>`

File: .github/ISSUE_TEMPLATE/b_goal.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: "\U0001f3c6 Goal"
 about: "Team Goal. Represents one specific delivery area from Azure Champions to the team. It will contain a set of :crown: Epics for this team."
-title: "\U0001f3c6 g-<team_alias>: <IaC|Backup|Monitoring|Disaster Recovery>"
+title: "\U0001f3c6 g-<team_alias>: <IaC|CI/CD|Backup|Monitoring|Disaster Recovery>"
```
Comment (nemanja-drobac) [inline .github/ISSUE_TEMPLATE/b_goal.md:4]:
```suggestion
title: "\U0001f3c6 g-<team_alia>: <IaC|CI/CD|Backup|Monitoring|Disaster Recovery>"
```

File: .github/ISSUE_TEMPLATE/e_task.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-about: 'Member of a Collection. Represents a single Pull Request or one manual operation. Use conventional commits with prefix t-act: feat|fix|chore(<environment>): <short name>'
+about: 'Member of a Collection. Represents a single Pull Request or one manual operation.'
 title: ''
 ---
 
-## Summary
+### Summary
 
-> DevOps link: none <!-- Example: AB#<item_number> -->
+> DevOps link: `none` <!-- Example: AB#<item_number> -->
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/e_task.md:9]: Could we undo these? ref #7 
```suggestion
> DevOps link: none <!-- Example: AB#<item_number> -->
```

File: .github/ISSUE_TEMPLATE/d_collection_maintenance.md [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
 
-> Remember the goal type of your tasks here. If any tasks related to another :trophy: Goal type, they should be inside that specific :card_index: Collection of tasks.
+> Remember the goal type of your tasks here. If any existing tasks are related to another :trophy: Goal type, they should be moved to respective :crown: Epic ->  inside that specific :card_index: maintenance collection.
 
-## Task Group(s)
+### Summary
 
-### <team_alias> tasks:
-- [ ] 
+> DevOps link: `none` <!-- Example: AB#<item_number> -->
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/d_collection_maintenance.md:15]: Could we undo these? ref #7 
```suggestion
> DevOps link: none <!-- Example: AB#<item_number> -->
```

File: .github/ISSUE_TEMPLATE/d_collection.md [hunk 4]
```diff
@@ -4,10 +4,20 @@ about: "Group(s) of tasks"
 title: "\U0001f4c7 c-<team_alias>:"
 ---
 
-## Summary
+### Summary
 
-DevOps link: none <!-- Example: AB#<item_number> -->
+> DevOps link: `none` <!-- Example: AB#<item_number> -->
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/d_collection.md:9]: Could we undo these? ref #7 
```suggestion
> DevOps link: none <!-- Example: AB#<item_number> -->
```

File: .github/ISSUE_TEMPLATE/c_epic.md [hunk 5]
```diff
@@ -5,17 +5,16 @@ title: "\U0001f451 e-<team_alias>:"
 version: "v1.0.1"
 ---
 
-## Summary
+### Summary
 
-DevOps link: none <!-- Example: AB#<item_number> -->
+> DevOps link: `none` <!-- Example: AB#<item_number> -->
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/c_epic.md:10]: Could we undo these? ref #7 
```suggestion
> DevOps link: none <!-- Example: AB#<item_number> -->
```

File: .github/ISSUE_TEMPLATE/d_collection_maintenance.md [hunk 6]
```diff
@@ -1,30 +1,29 @@
...
 about: "Group(s) of tasks"
-title: "\U0001f4c7 c-<team_alias>: <goal> maintenance 2024 Q1"
+title: "\U0001f4c7 mc-<team_alias>:"
 ---
 
-## Summary
+## Purpose
 
-This is a maintenance collection that contains one ore more groups of tasks. Remember that this is a maintenance collection that needs to be closed inside the current quarter of this year. If some of the tasks here cannot be closed, please transfer the task into the next quarter :card_index: Collection for this project.
+This is a maintenance collection that contains one ore more groups of tasks. Remember that this is a maintenance collection that needs to be closed inside the current quarter of this year. If some of the tasks here cannot be closed, please transfer the task into the next quarter :card_index: maintenance collection for this project.
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/d_collection_maintenance.md:9]:
```suggestion
This is a maintenance collection that contains one or more groups of tasks. Remember that this is a maintenance collection that needs to be closed inside the current quarter of this year. If some of the tasks here cannot be closed, please transfer the task into the next quarter :card_index: maintenance collection for this project.
```

File: .github/ISSUE_TEMPLATE/e_task.md [hunk 7]
```diff
@@ -1,15 +1,21 @@
...
+#### Acceptance Criteria
+
+- None
 
 ## To do
 
 - [ ]
 
-<!-- Please remember to create branch from this issue to make sure Pull Request closes this issue automatically. -->
+> Please remember to create branch from this issue to make sure Pull Request closes this issue automatically. Use conventional commits. [Link](https://www.conventionalcommits.org/en/v1.0.0/)
```
Comment (vildeaar) [inline .github/ISSUE_TEMPLATE/e_task.md:21]: Should we add link to cyclops documentation regarding this as well?

Comment (vildeaar) [reply .github/ISSUE_TEMPLATE/e_task.md:21]: this link - [cyclops docs](https://urban-waffle-59ea765a.pages.github.io/issues/#commit-messages)

Comment (nemanja-drobac) [reply .github/ISSUE_TEMPLATE/e_task.md:21]: yes, I will fix that

File: .github/ISSUE_TEMPLATE/e_task.md (hunk 2 above)
Comment (nemanja-drobac) [reply .github/ISSUE_TEMPLATE/e_task.md]: Fixed in latest commits.

File: .github/ISSUE_TEMPLATE/c_epic.md (hunk 5 above)
Comment (nemanja-drobac) [reply .github/ISSUE_TEMPLATE/c_epic.md]: Fixed in latest commits.

File: .github/ISSUE_TEMPLATE/e_task.md (hunk 7 above)
Comment (nemanja-drobac) [reply .github/ISSUE_TEMPLATE/e_task.md]: Fixed in latest commits.

File: .github/ISSUE_TEMPLATE/b_goal.md
[the remark this answers is not in the document]
Comment (nemanja-drobac) [reply .github/ISSUE_TEMPLATE/b_goal.md:1]: Fixed in latest commits.

**Review by eikeskogen — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/c_epic_maintenance.md [hunk 8]
```diff
@@ -1,41 +1,27 @@
 ---
-name: "\U0001f451 Epic Maintenance"
+name: "\U0001f451 Maintenance Epic"
 about: "Member of a specific Team goal. It consists of one or more task-collections"
-title: "\U0001f451 e-<team_alias>: Maintenance 2024"
+title: "\U0001f451 e-<team_alias>: <goal_type> Maintenance 2024 Q1"
```
Comment (eikeskogen) [inline .github/ISSUE_TEMPLATE/c_epic_maintenance.md:4]:
```suggestion
title: "\U0001f451 me-<team_alias>: <goal_type> Maintenance 2024 Q1"
```

**Review by eikeskogen — APPROVED** [review]

**Pull request merged**